### ISSUE TITLE
Toggles not visible in standalone client

### ISSUE THREAD

**Issue by wrymn** [the-mr-matt/DebugUISystem#1]
Toggles are not working when playing built standalone client.

By toggles I mean drawing toggle via **GUILayout.Toggle()** into custom window

**Comment by JudahMantell** [comment]
I tested this myself and they do work, but the actual UI images just are invisible.  Still weird.  I hope a fix is possible! :)

**Comment by wrymn** [comment]
@MidnightCoffeeInc yeah thats what I had in mind, without the toggle checkbox UI its not really usable, since demo panel really shows off at standalone build where you need to debug/cheat stuff for the game, since you dont have editor
